yoyodyne/models/modules/base.py

```diff
@@ -13,10 +12,12 @@
 
 @dataclasses.dataclass
 class ModuleOutput:
-    """For tracking outputs of forward passes over varying architectures."""
+    """Output for forward passes."""
 
     output: torch.Tensor
-    hiddens: Optional[Tuple[torch.Tensor, torch.Tensor]] = None
+    hiddens: Optional[
```

Comment:
This overloading of the hiddens property strikes me as a case for just creating a new module output class.

Reply:
I think a class called `Hiddens` is the right solution in the future, yes.